config/default/translations/messages-en.properties

```diff
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.pregnancy_danger_sign.danger_signs.danger_signs_note = Patient to monitor these danger signs throughout the pregnancy.
+report.pregnancy_danger_sign.danger_signs.danger_signs_question_note = Has any of these danger signs
+report.pregnancy_danger_sign.danger_signs.vaginal_bleeding = Vaginal Bleeding
+report.pregnancy_danger_sign.danger_signs.fits = Fits
+report.pregnancy_danger_sign.danger_signs.severe_abdominal_pain = Severe abdominal pain
+report.pregnancy_danger_sign.danger_signs.severe_headache = Severe headache
+report.pregnancy_danger_sign.danger_signs.very_pale = Very pale
+report.pregnancy_danger_sign.danger_signs.fever = Fever
+report.pregnancy_danger_sign.danger_signs.reduced_or_no_fetal_movements = Reduced or no fetal movements
+report.pregnancy_danger_sign.danger_signs.breaking_water = Breaking of water
+report.pregnancy_danger_sign.danger_signs.easily_tired = Getting tired easily
+report.pregnancy_danger_sign.danger_signs.face_hand_swelling = Swelling of face and hands
+report.pregnancy_danger_sign.danger_signs.breathlessness = Breathlessness
+report.pregnancy_danger_sign.danger_signs.custom_translations = Custom translations
+report.pregnancy_danger_sign.danger_signs.custom_woman_label_translator = Woman Label translator
+report.pregnancy_danger_sign.danger_signs.custom_woman_label = Woman label
+report.pregnancy_danger_sign.danger_signs.custom_woman_start_label_translator = Woman Start Label translator
+report.pregnancy_danger_sign.danger_signs.custom_woman_start_label = Woman Start label
+
+report.pregnancy_home_visit.context_vars =  Context Variables
+report.pregnancy_home_visit.context_vars.lmp_date_8601_ctx = LMP
+report.pregnancy_home_visit.context_vars.lmp_method_approx_ctx = LMP  Approximate
+report.pregnancy_home_visit.context_vars.pregnancy_follow_up_date_recent_ctx = Recent Pregnancy Follow Up Date
+report.pregnancy_home_visit.context_vars.risk_factors_ctx = Risk Factors
+report.pregnancy_home_visit.context_vars.risk_factor_extra_ctx = Extra risk factors
+report.pregnancy_home_visit.context_vars.hiv_tested_ctx =  Tested for HIV
+report.pregnancy_home_visit.context_vars.tt_received_ctx = Recieved TT
+report.pregnancy_home_visit.context_vars.deworming_med_received_ctx = Recieved Deworming Medications
+report.pregnancy_home_visit.context_vars.edd_ctx = EDD
+report.pregnancy_home_visit.context_vars.weeks_since_lmp_rounded_ctx = Weeks since LMP
+report.pregnancy_home_visit.context_vars.pregnancy_uuid_ctx = Pregnancy UUID
+
+report.pregnancy_home_visit.pregnancy_summary = Pregnancy Summary
+report.pregnancy_home_visit.pregnancy_summary.current_weeks_pregnant = Current Weeks Pregnant
+report.pregnancy_home_visit.pregnancy_summary.current_weeks_pregnant_unknown = Current Weeks Pregnant
+report.pregnancy_home_visit.pregnancy_summary.current_edd = Expected Date of Delivery(EDD)
+report.pregnancy_home_visit.pregnancy_summary.current_edd_unknown = Expected Date of Delivery(EDD)
```

Comment:
Similar to above case of two labels having the same label... this may be intended but calling out for review.